package.json

```diff
@@ -13,7 +15,7 @@
     "lint:css": "stylelint packages/**/src/*.js packages/**/theme/**/*-styles.js",
     "lint:js": "eslint --ext .js,.ts *.js scripts packages integration",
     "lint:types": "tsc",
-    "postinstall": "patch-package",
+    "postinstall": "patch-package && npm run build",
```

Comment:
Would it be enough to only run this when publishing components, like we do with web-types etc.? When switching to older branches you end up with a lot of untracked files otherwise, and since it's automatic, is there any benefit to having these files during development?